docs/user/03-traces.md

```diff
@@ -274,9 +276,10 @@ stringData:
   endpoint: https://backend.example.com:4317
   user: myUser
   password: XXX
-  token: Bearer YYY
+  token: YYY
 ```
 
+The value of the token can be stored in the referenced secret without any prefix or scheme (in this example, the token has the prefix Bearer), and it can be configured in the headers section of the TracePipeline.
```

Comment:
```suggestion
The value of the token can be stored in the referenced Secret without any prefix or scheme, and it can be configured in the headers section of the TracePipeline. In this example, the token has the prefix Bearer.
```

Reply:
Just to confirm and be on the safe side, do you mean "scheme" or "schema"?